.github/workflows/issue-opened.yml

```diff
@@ -75,19 +77,21 @@ jobs:
                     labelExists = true;
                   } catch {}
 
-                  if (labelExists) {
-                    // Check if it's an unsupported major
-                    const { ElectronVersions } = await import('${{ github.workspace }}/node_modules/@electron/fiddle-core/dist/index.js');
-                    const versions = await ElectronVersions.create(undefined, { ignoreCache: true });
+                  const { ElectronVersions } = await import('${{ github.workspace }}/node_modules/@electron/fiddle-core/dist/index.js');
+                  const electronVersions = await ElectronVersions.create(undefined, { ignoreCache: true });
+                  const validVersions = [...electronVersions.supportedMajors, ...electronVersions.prereleaseMajors];
 
-                    const validVersions = [...versions.supportedMajors, ...versions.prereleaseMajors];
-                    if (validVersions.includes(major)) {
+                  if (validVersions.includes(major)) {
+                    if (labelExists) {
                       labels.push(versionLabel);
                     }
+                  } else {
+                    hasUnsupportedVersion = true;
```

Comment:
This introduces a new bug: Let's say someone reports an issue with the following version string: `36.4.0, 35.1.1, 28.0.0`. Then the bot will see one unsupported version and throw out the EOL message. We'd want to do that only if all versions are unsupported.

We might want to invert the logic: initialize a variable `hasSupportedVersion = false`, set it to `true` if `validVersions.includes(major)` and show the message if `!hasSupportedVersion`.

Reply:
Seems reasonable.